package.json

```diff
@@ -14,6 +14,7 @@
     "@hypothesis/frontend-testing": "^1.2.2",
     "@rollup/plugin-babel": "^6.0.0",
     "@rollup/plugin-commonjs": "^25.0.0",
+    "@rollup/plugin-dynamic-import-vars": "^2.1.2",
```

Comment:
This plugin allows dynamic imports where the path is also dynamic, like `import(`../examples/${somefile}.tsx`).

It has the limitation that it scans for potential files when building the first time, but if new files are added to the examples directory, you need to restart the dev server.

Reply:
> It has the limitation that it scans for potential files when building the first time, but if new files are added to the examples directory, you need to restart the dev server.

From reading the [source](https://github.com/rollup/plugins/blob/master/packages/dynamic-import-vars/src/index.js) for the plugin, it looks to me like modifying the source code of the file which contains the import statement should also work, as it executes the glob as part of processing such an import.

Reply:
Oh, good point.